### ISSUE TITLE
Add option to enter two characters instead

### ISSUE THREAD

**Issue by Myzel394** [schasse/tmux-jump#42]
So [leap](https://github.com/ggandor/leap.nvim) is a plugin for neovim which behaves very similar but expects the user to enter two characters instead.

Using two characters limits down the amount of characters drastically and thus makes it easier to jump to a given word.
Therefore I propose to add an option that allows you to enter two characters instead to jump to your word

**Comment by schasse** [comment]
Thank you for your suggestion and it's a great idea. However, due to my current workload, I'm unable to implement this feature at the moment. I would greatly appreciate it if you could contribute by creating a pull request for this feature.

**Comment by Myzel394** [comment]
I can take a look and see if I can figure that out on my own. I scanned briefly over the code and it seems as if the main logic is in this ruby file (https://github.com/schasse/tmux-jump/blob/master/scripts/tmux-jump.rb) isn't it?

**Comment by schasse** [comment]
Yes, it the main logic for the tool is indeed in the Ruby file you mentioned
